src/parcsr_ls/par_mgr_interp.c

```diff
@@ -2096,167 +2121,6 @@ hypre_MGRBuildInterpApproximateInverse(hypre_ParCSRMatrix   *A,
    return hypre_error_flag;
 }
 
-/*--------------------------------------------------------------------------
- * hypre_MGRGetAcfCPR
- *--------------------------------------------------------------------------*/
-
-HYPRE_Int
-hypre_MGRGetAcfCPR(hypre_ParCSRMatrix     *A,
```

Comment:
Has this been consolidated into the MGRBuildR function?

Reply:
This function does two things:
1. Computes A_CF
2. Truncates A_CF to block (rectangular) diagonal.

We already have other code that does 1 and 2 separately. In addition, when calling this function in MGR, we already have knowledge about A_CF (it's formed right at the beginning of each level)

So, I removed this function and replaced it with a call to `hypre_MGRTruncateAcfCPR`, which does 2.

The final code is better performant (we do less extra computation) and yields the same result